src/html/page_template.rs

```diff
@@ -62,7 +65,7 @@ fn dump_non_container_tag(tag_name: &str, attributes: &BTreeMap<String, String>)
         .iter()
         .map(|(key, value)| format!(" {}=\"{}\"", key.escape_html(), value.escape_html()))
```

Comment:
Clippy suggests this approach in here instead: https://rust-lang.github.io/rust-clippy/master/index.html#/format_collect

Reply:
@Dudu7352 did you read what I linked?